### ISSUE TITLE
HEAL function have unlimited range

### ISSUE THREAD

**Issue by rlex** [DUWS-R-Team/DUWS-R#61]
Heal function in DUWS Modified have unlimited range. Technically, you can play leader and send your troops all over the map, while sitting at base and spamming HEAL (2CP) support. Probably should only handle troops within ~100m radius.

**Comment by fritogotlayed** [comment]
@rlex We are still not based off of DUWS Modified if I am understanding your issue correctly. The PBO you linked in #57 has not been pulled into DUWS-R. This issue doesn't affect DUWS-R that I know of since that feature was not in the original DUWS from Kibot.

**Comment by rlex** [comment]
I will be pulling that changes in DUWS-R if you're not against it. 
Heal function as well, so bug will still be here.

**Comment by fritogotlayed** [comment]
I'm not against it right now b/c I've never played your version honestly. :smile: The only real thing I'm still apprehensive of is having a bunch of non vanilla ArmA vehicles and the such in. I may just be broken but I kind of like the vanilla assets so I want to keep that support there. The group of people I play DUWS-R / ArmA 3 with enjoy the core experience with Arma 3. We each have one or two local-only mods but we're still pretty vanilla across the board. 

All that said I'd love to for DUWS-R to be the upstream for many of those variations. The ease of source sharing on GitHub was one of the big reasons I wanted to put the source out here. That and I really feel that I can't think of everything that would really improve the core mission game play. 

I guess that was a really long way to say that I just don't have any experience with the version you linked and I really need to go check it out. If you're familiar with the Gallup strengths, basically my Deliberative-Analytical-Intellection is showing, lol.

[Deliberative](http://www.gallup.com/businessjournal/658/deliberative.aspx)
[Analytical](http://www.gallup.com/businessjournal/631/analytical.aspx)
[Intellection](http://www.gallup.com/businessjournal/691/intellection.aspx)
[Learner](http://www.gallup.com/businessjournal/694/learner.aspx)
[Relator](http://www.gallup.com/businessjournal/703/relator.aspx)

**Comment by rlex** [comment]
No, no. I will not add any modded vehicles in PRs. If i need them, i add them locally.
I still have 3GB of arma mods to play with my friends, but that's mostly weapons, uniforms, agm and so on

**Comment by fritogotlayed** [comment]
:-) All good. I was not trying to imply you would. I was just trying to articulate my vision for DUWS-R.
